README.md

```diff
@@ -279,7 +280,7 @@ basePath: /path/to/sql
 
 sql:
   - schema/<schema_name>.sql
-  - function/<schema_name>/<function_name>.sql
+  - function/<schema_name>/<function_name>.sql.dropped #While syncing this will only be dropped, not upped.
```

Comment:
- function/<schema_name>/<function_name>.sql.dropped # While synchronizing this object will only be dropped, not created